dm/master/server_test.go

```diff
@@ -2868,3 +2870,49 @@ func (t *testMaster) TestDashboardAddress(c *check.C) {
 	c.Assert(err, check.IsNil)
 	c.Assert(string(content), check.Matches, "[\\s\\S]*Web UI enabled[\\s\\S]*")
 }
+
+func (t *testMaster) TestGetNewestMeta(*check.C) {
```

Comment:
```suggestion
func TestGetNewestMeta(t *testing.T) {
```

Reply:
use testSuite to avoid concurrency flaky test